requirements/lite.txt

```diff
@@ -0,0 +1,2 @@
+accelerate
```

Comment:
Shall we add version control here? @lvhan028

Reply:
Problem solved after I updated accelerate to the latest version